src/r-packages/NOTES.md

```diff
@@ -0,0 +1,56 @@
+<!-- markdownlint-disable MD041 -->
+
+## System Requirements
+
+Supports `linux/amd64` and `linux/arm64` platforms with R installed.
+
+This Feature does not install R.
+Please use this with an R-installed image (e.g. [`ghcr.io/rocker-org/devcontainer/r-ver`](https://rocker-project.org/images/devcontainer/images.html))
+or this should be installed after installing Features that installs R
+(e.g. [`ghcr.io/rocker-org/devcontainer-features/r-apt`](https://github.com/rocker-org/devcontainer-features/tree/main/src/r-apt),
+[`ghcr.io/rocker-org/devcontainer-features/r-rig`](https://github.com/rocker-org/devcontainer-features/tree/main/src/r-rig)).
+
+```json
+"features": {
+    "ghcr.io/rocker-org/devcontainer-features/r-rig:1": {},
+    "ghcr.io/rocker-org/devcontainer-features/r-packages:1": {
+        "packages": "cli,rlang"
+    }
+},
+"overrideFeatureInstallOrder": [
+    "ghcr.io/rocker-org/devcontainer-features/r-rig"
+]
```

Comment:
Once the CLI bug is fixed, these lines are not needed, so it would be better to remove these so as not to cause confusion.

```suggestion
}
```